prompt.go

```diff
@@ -49,6 +60,49 @@ func (i *InitActionCmd) setParams(dirs []string) error {
 			}
 		}
 	}
+	if i.ConfigPath == "" && len(configs) > 0 {
+		prompt := promptui.Select{
+			Label: "Choose atlas.hcl config file path to use",
+			Items: configs,
+			Stdin: i.stdin,
+		}
+		if _, i.ConfigPath, err = prompt.Run(); err != nil {
+			return err
+		}
+		content, err := repo.ReadContent(ctx, i.ConfigPath)
+		if err != nil {
+			return err
+		}
+		file, diags := hclparse.NewParser().ParseHCL([]byte(content), "atlas.hcl")
+		if len(diags) > 0 {
+			return fmt.Errorf("failed to parse %s: %s", i.ConfigPath, diags.Error())
+		}
+		var envs = make(map[string]bool)
+		for _, b := range file.Body.(*hclsyntax.Body).Blocks {
+			if b.Type == "env" {
+				_, hasDev := b.Body.Attributes["dev"]
+				envs[b.Labels[0]] = hasDev
+			}
+		}
+		switch {
+		case len(envs) == 0:
+			return fmt.Errorf("no env blocks found in %s", i.ConfigPath)
```

Comment:
`env` is not required. You can define `lint` and `diff` as top-level blocks and pass the config using the `-c` flag => if no env was selected, the user still needs to have a dev-database in its yaml file.